tests/hci_test.py

```diff
@@ -215,12 +215,13 @@ def test_HCI_Command():
 
 # -----------------------------------------------------------------------------
 def test_HCI_PIN_Code_Request_Reply_Command():
+    code = b'1234'
```

Comment:
should be `b'001234'`

Reply:
Per our discussion, updated with padding zero bytes after the pin code